libkbfs/key_manager_test.go

```diff
@@ -1077,6 +1077,92 @@ func TestKeyManagerReaderRekey(t *testing.T) {
 	}
 }
 
+func TestKeyManagerReaderRekeyAndRevoke(t *testing.T) {
```

Comment:
I tried that at first, but it didn't work because the second Rekey returns an error [here](https://github.com/keybase/kbfs/blob/master/libkbfs/key_manager.go#L573).  To get into this state, the reader needs a revoked device.

(Side note: `ReadAccessError` is probably not the right error for the case I linked to above.)